### ISSUE TITLE
FEAT(⭐): Activate Claims Gate

### ISSUE THREAD

**Issue by alfl** [kubelt/kubelt#622]
# Why

Kubelt is a platform for building decentralized applications. Starbase is the dapp that allows developers to configure and launch _their_ dapps.

Similarly to 3iD, we want to gate access to Starbase so that we can continuously deploy to production while controlling rollout.

# What

- Decide on a name for the access claim, for example `starbase.enter`.
- Implement the early access gate screen flow from 3iD to gate access to the Starbase dapp.
- When a user authenticates with Starbase, the claims associated with their wallet address are queried from Oort to determine whether they are authorized (i.e., if their claims include `starbase.enter`).
- When a user manually navigates to a sub-page for the dapp (without loss of generality, for example, https://starbase.kubelt.com/config) we ensure they have access to the sub-page (potentially directing the user through the authentication and authorization flow again if they are disconnected/logged out), and then routed to the sub-page.
- Access to Oort routes will be gated by the same claim (e.g., `starbase.enter`).

# How

Please ticket your approach and discuss.

# Notes

- N/A

**Comment by alfl** [comment]
cc @szkl -- adding a new claim here. We should discuss making the claim types richer so that we can grant a set of capabilities to an application without updating route handlers. Will ticket.

**Comment by alfl** [comment]
cc @Cosmin-Parvulescu -- the intent here is that Starbase is gated the same way as 3iD. Please support Drew on hitting feature parity. Could you also take a look at the **What** section, above, and see if that's a clear description of the flow.

**Comment by alfl** [comment]
Moved to SDK team. Lower priority than minting.
